simulation/traffic_simulator/src/entity/entity_manager.cpp

```diff
@@ -88,6 +88,15 @@ bool EntityManager::checkCollision(const std::string & name0, const std::string
   return traffic_simulator::math::checkCollision2D(status0->pose, bbox0, status1->pose, bbox1);
 }
 
+const visualization_msgs::msg::MarkerArray EntityManager::getDebugMarker() const
+{
+  visualization_msgs::msg::MarkerArray marker;
+  for (const auto & entity : entities_) {
+    entity.second->appendDebugMarker(marker);
+  }
+  return marker;
+}
```

Comment:
In general, a getter is simply a member function that returns a reference to a data member.
This function that builds and returns new data is not getter.
Rename this member function to `makeDebugMarker`.